app/services/suggesters/tags.rb

```diff
@@ -9,12 +9,17 @@ def initialize(tags:, except: [], limit: 10, models: [Group, List, Resource])
 
     def suggest
       load_es_params
-      except_records if @except
-      Elasticsearch::Model.search(@es_params, @models).records
+      records
     end
 
     private
 
+    def records
```

Comment:
This is used to load the suggestions on the search page or on the resource page. It will loads records based on the passed tags (and the other options). The `except` is only used for the resource page, in order to remove the current resource from the list of suggestions.